src/analysis/max_min_driving_force.jl

```diff
@@ -55,56 +55,95 @@ optimization problems.
 """
 function max_min_driving_force(
     model::MetabolicModel,
-    gibbs_free_energies::Dict{String,Float64},
+    metabolite_gibbs_free_energies::Dict{String,Float64},
     optimizer;
-    ignore_metabolites::Vector{String} = ["h", "h2o"],
+    flux_solution::Dict{String,Float64} = Dict{String,Float64}(),
+    reaction_gibbs_free_energy_adjustments::Dict{String,Float64} = Dict{String,Float64}(),
+    metabolite_gibbs_free_energy_errors::Dict{String,Float64} = Dict{String,Float64}(),
+    proton_ids::Vector{String} = ["h_c", "h_e"],
+    water_ids::Vector{String} = ["h2o_c", "h2o_e"],
     constant_concentrations::Dict{String,Float64} = Dict{String,Float64}(),
     concentration_ratios::Dict{Tuple{String,String},Float64} = Dict{
         Tuple{String,String},
         Float64,
     }(),
+    ignore_reactions::Vector{String} = Vector{String}(),
     concentration_lb = 1e-6,
     concentration_ub = 10e-3,
     T = _constants.T,
     R = _constants.R,
+    metabolite_gibbs_free_energy_error_fallback_fraction = 0.01,
     modifications = [],
+    negligible_flux = 1e-6,
```

Comment:
is this tolerance? (might be defaulted from constants)

Also, what about decoupling it to something like
```
remove_negligible_fluxes(flux_solution, 0.01)
```

Reply:
yep, a tolerance. Hmm maybe the user should be doing this... will require them to actually read the docs though